tools/README

```diff
@@ -103,6 +103,33 @@ Examples:
   $ sof-logger -l ldc_file -i trace_dump -o out_file -c 19.9
 
 
+coredump-reader
```

Comment:
_sof-coredump-reader_ to be consistent with other sections.

Reply:
ok, changed